crates/frontend/src/components/table.rs

```diff
@@ -29,8 +30,37 @@ pub fn table(
                             .iter()
                             .filter(|column| !column.hidden)
                             .map(|column| {
-                                view! {
-                                    <th class="uppercase">{&column.name.replace('_', " ")}</th>
+                                let column_name = (&column.name).replace('_', " ");
+                                match column.sortable.clone() {
+                                    types::ColumnSortable::Yes {
+                                        sort_on: _,
+                                        sort_fn,
+                                        sort_by,
+                                        current,
+                                    } => {
+                                        view! {
+                                            <th
+                                                class="uppercase"
+                                                style="cursor: pointer"
+                                                on:click=move |_| sort_fn.call(())
+                                            >
+                                                {column_name}
+                                                    {
+                                                        if current {
```

Comment:
current checks if this column is currently being sorted on, if it is - it will only show the up arrow or down arrow. If it is not, it will show both arrows